PowerPointLabs/PowerPointLabs/ELearningLab/Utility/ShapeUtility.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.Office.Core;
+using Microsoft.Office.Interop.PowerPoint;
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.Models;
+using PowerPointLabs.TextCollection;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+
+namespace PowerPointLabs.ELearningLab.Utility
+{
+    public class ShapeUtility
+    {
+#pragma warning disable 0618
```

Comment:
`SlideWith` and `SlideHeight` information are retrieved from `PowerPointCurrentPresentationInfo` when creating shapes. Since ShapeUtility does not inherit from ActionFramework and it is meant to be a utility class so that other services class can use this class and create shapes to slide, I find it a bit hard to use ActionFramework in this case.